python-sdk/src/astro/dataframes/load_options.py

```diff
@@ -0,0 +1,32 @@
+from __future__ import annotations
+
+from attr import define
+from pandas._typing import DtypeArg
+
+from astro.options import LoadOptions
+
+
+@define
+class PandasLoadOptions(LoadOptions):
+    pass
+
+
+@define
+class CsvLoadOption(PandasLoadOptions):
+    delimiter: str | None = None
+    dtype: DtypeArg | None = None
+
+
+@define
+class JsonLoadOption(PandasLoadOptions):
```

Comment:
I noticed some options we use `Options` and others `Option`. I suggest we stick to `Options`, since these classes will usually contain a group of configurations

Reply:
cc: @pankajastro

Reply:
I too like this suggestion, we should stick with `options`